spec/models/utils/nav/customization_parser_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Utils::Nav::CustomizationParser do
+  it "No customizations" do
+    parser = Utils::Nav::CustomizationParser.new(nil)
+    expect(parser.parse).to eq({})
+  end
+  it "One item" do
+    parser = Utils::Nav::CustomizationParser.new("[Google](http://www.google.com)")
+    expect(parser.parse).to eq("Google" => "http://www.google.com")
+  end
+  it "Two items" do
+    parser = Utils::Nav::CustomizationParser.new("[Google](http://www.google.com)[Yahoo](http://www.yahoo.com)")
+    expect(parser.parse).to eq("Google" => "http://www.google.com", "Yahoo" => "http://www.yahoo.com")
+  end
```

Comment:
In this multiple link case, can you also please test that the hash keys are in the right order? So `expect(result.keys).to eq(...)`